<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>522</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="ComboFrame" name="frameCombos">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionEnableDisableBeeftext"/>
    <addaction name="separator"/>
    <addaction name="actionShowPreferencesDialog"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionGettingStarted"/>
    <addaction name="actionVisitBeeftextWiki"/>
    <addaction name="actionShowReleaseNotes"/>
    <addaction name="separator"/>
    <addaction name="actionShowAboutDialog"/>
   </widget>
   <widget class="QMenu" name="menu_Advanced">
    <property name="title">
     <string>&amp;Advanced</string>
    </property>
    <addaction name="actionOpenLogFile"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Advanced"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit Beeftext</string>
   </property>
   <property name="iconText">
    <string notr="true">Exit &lt;insertAppNameHere&gt;</string>
   </property>
   <property name="toolTip">
    <string>Exit Beeftext</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionVisitBeeftextWiki">
   <property name="text">
    <string>&amp;Beeftext Wiki</string>
   </property>
   <property name="shortcut">
    <string>Shift+F1</string>
   </property>
  </action>
  <action name="actionShowAboutDialog">
   <property name="text">
    <string>&amp;About Beeftext</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionEnableDisableBeeftext">
   <property name="text">
    <string notr="true">#EnableDisableBeeftext#</string>
   </property>
   <property name="iconText">
    <string notr="true">#EnableDisableBeeftext#</string>
   </property>
   <property name="toolTip">
    <string notr="true">#EnableDisableBeeftext#</string>
   </property>
  </action>
  <action name="actionShowPreferencesDialog">
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="actionOpenLogFile">
   <property name="text">
    <string>Open &amp;Log File</string>
   </property>
   <property name="toolTip">
    <string>Open the log file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="actionGettingStarted">
   <property name="text">
    <string>&amp;Getting Started</string>
   </property>
   <property name="toolTip">
    <string>Getting Started</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F1</string>
   </property>
  </action>
  <action name="actionShowReleaseNotes">
   <property name="text">
    <string>&amp;Release Notes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ComboFrame</class>
   <extends>QFrame</extends>
   <header>Combo/ComboFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Beeftext.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEnableDisableBeeftext</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionEnableDisableBeeftext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowAboutDialog</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionShowAboutDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowPreferencesDialog</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionShowPreferencesDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenLogFile</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionOpenLogFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onActionExit()</slot>
  <slot>onActionShowMainWindow()</slot>
  <slot>onActionEnableDisableBeeftext()</slot>
  <slot>onActionShowAboutDialog()</slot>
  <slot>onActionShowPreferencesDialog()</slot>
  <slot>onActionOpenLogFile()</slot>
 </slots>
</ui>
